### PR TITLE
Guitar bonk

### PR DESCRIPTION
**Pull request by flappybatpal**
[QOL]

## About the PR 
Adds funny sound effects when beating someone with a guitar.

Base sound: https://freesound.org/people/Mystovski/sounds/201111/
License - https://creativecommons.org/publicdomain/zero/1.0/
Changes made - Duration trimmed, pops removed, volume fade out, compressed and converted to .ogg

Sound 1: https://freesound.org/people/jay_mar/sounds/566172/
License - https://creativecommons.org/publicdomain/zero/1.0/
Changes made - Duration trimmed, volume fade out, mixed with base sound, compressed and converted to .ogg

Sound 2: https://freesound.org/people/Ruvhjnvdkvndjine/sounds/569289/
License - https://creativecommons.org/publicdomain/zero/1.0/
Changes made - Duration trimmed, volume fade out, mixed with base sound, compressed and converted to .ogg

Sound 3: https://freesound.org/people/chris.t/sounds/452886/
License - https://creativecommons.org/publicdomain/zero/1.0/
Changes made - Duration trimmed, volume fade out, mixed with base sound, compressed and converted to .ogg

https://user-images.githubusercontent.com/70909958/121661860-8df0a780-ca9c-11eb-849d-aa082232218c.mp4

## Why's this needed? 
Immersion.

### CODE REVIEW

**Review by virvatuli-gh — CHANGES_REQUESTED** [review]
These sounds are far too long for attack audio imo, which is something that can be spammed fairly quickly. It'll result in too much overlapping audio cues when someone is rapidly attacking someone else. Aside from the length, their filesizes are rather large (which may be solved by cutting them down, but could also be compressed better on top of that).

Additionally, make sure you are actually reading the licensing for the sounds you're using, two of the files you chose are not CC0 and require additional things outside of just linking them in the PR. This is why I'd prefer if additional sound files are CC0 (which have no attribution) or if the file is made for the codebase itself, as it bypasses the need of creating a file in the codebase that has a hundred attribution links in it.

**Comment by flappybatpal** [comment]
Actually set the CC0 filter and picked all sounds to be CC0.

Changed all sounds to 1 second length with volume fade out. Impact sounds are like 0.5-1.0 second, with the fade out should be better than say the circular saw.

Changed the .ogg quality to 0.5, now <15KB.

File: .vscode/settings.json [hunk 1]
```diff
@@ -25,11 +25,8 @@
 		"editor.rulers": [150]
 	},
 	// Lets you preview .dmi files in the editor
-	"workbench.editorAssociations": [
```
Comment (virvatuli-gh) [inline .vscode/settings.json:28]: what's up with this unrelated file change?

Comment (flappybatpal) [reply .vscode/settings.json:28]: I have no idea but vscode is forcing it into any branch I try and edit.

Comment (ZeWaka) [reply .vscode/settings.json:28]: I'll fix this.

**Review by virvatuli-gh — APPROVED** [review]
